spec/core_functions/color/legacy.hrx

```diff
@@ -0,0 +1,182 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+===================================================================
+<===> rgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(rgb(0 255 0))}
+
+<===> rgb/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> hwb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(hwb(0, 50%, 0%))}
+
+<===> hwb/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> hsl/input.scss
+@use "sass:color";
+a {b: color.is-legacy(hsl(110, 31%, 32%))}
+
+<===> hsl/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> srgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(srgb(115, 20, 210))}
```

Comment:
@mirisuzanne @dvdherron I've not really come across a lot with srgb used in a css notation like this before, but would it be more like:

```
a {b: color.is-legacy(color(srgb 0.45098 0.07843 0.823530))}
```

Reply:
https://css.oddbird.net/sass/color-spaces/proposal/#predefined-color-spaces
> 
> ‘Predefined color spaces’ can be described using the color() function.
> The predefined RGB spaces are:
> 
> - srgb
> - srgb-linear
> - display-p3
> - a98-rgb
> - prophoto-rgb
> - rec2020